kornia/enhance/normalize.py

```diff
@@ -184,3 +193,44 @@ def denormalize(
     out: torch.Tensor = (data * std) + mean
 
     return out
+
+
+def normalize_min_max(x: torch.Tensor, a: float = 0., b: float = 1., eps: float = 1e-6) -> torch.Tensor:
```

Comment:
Should we allow (B, ) for the a and b here? Also, replace a, b names with min_val, max_val?

Reply:
we need to investigate more to find a workaround since JIT is not compatible with Union types. @ducha-aiki @pmeier are you aware of anything about it ?

Reply:
@edgarriba In `torchvision` we are simply ignoring the errors that arise by not using `Union` for now. See https://github.com/pytorch/vision/pull/2668#discussion_r487095766 for an example.

Reply:
this might a common error if we want to adopt scripted operators across the whole library since we assume Union type everywhere. I guess one of the drawbacks of python for not being able to overload functions